### PR TITLE
Mil-17_sass_styling

### PR DESCRIPTION
**Pull request by Margo1212**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "start": "parcel --public-url ./ --no-hmr",
     "format": "prettier --write \"src/**/*.js\"",
     "lint": "eslint --quiet \"src/**/*.js\"",
-    "compile:sass": "node-sass sass/main.scss css/style.css -w"
+    "compile:sass": "node-sass src/sass/main.scss src/css/style.css -w"
```
Comment (matizaj) [inline package.json:12]: good point, I forgot to specify the path.

File: src/sass/main.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@import "utils/variables";
+@import "utils/normilize";
```
Comment (matizaj) [inline src/sass/main.scss]: looks like there is a typo. should be 'normalize' instead of 'normilize'. rename this file.

File: src/sass/main.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+@import "utils/mixins";
+@import "layout/body";
```
Comment (matizaj) [inline src/sass/main.scss]: you should add new empty line (hit enter)

File: src/sass/utils/normilize.scss [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+  display: none;
+}
+
+/**
+ * Add the correct display in IE 10.
+ */
+
+[hidden] {
+  display: none;
+}
```
Comment (matizaj) [inline src/sass/utils/normilize.scss]: you should add new empty line (hit enter)

**Review by matizaj — CHANGES_REQUESTED** [review]

File: src/sass/main.scss (hunk 2 above)
Comment (Margo1212) [reply src/sass/main.scss]: ok, thank you!

Comment (matizaj) [reply src/sass/main.scss]: you rename file normilize.scss to normalize.scss it's good but you also should name when you import this file in main.scss.
Second line of this file: '@import "utils/normilize"' -> you should also change name to normalize in this line.

Comment (Margo1212) [reply src/sass/main.scss]: Oh, I changed this import, but didn't commit it. Thank you for review!

**Review by matizaj — APPROVED** [review]
ok

**Pull request merged**